### DISCUSSION TITLE
Nouveau site

### DISCUSSION THREAD

**Discussion by Tritri1029** [Kodi-vStream/venom-xbmc-addons#4044] [Idées]
Malheureusement je n'ai pas le talent pour crée le code moi même mais je crois que papaflix ou Tirexo et ressortie sur un autre nom donc je poste ce message si quelqu'un aurait le temps de faire la possibilité d'utilisation de darkino.net (le nouveau papaflix)

**Comment by sizanic** [comment]
Bonjour,
Par contre je pense que tu dois avoir assez de talent pour faire une recherche parmi les sujets existants pour vérifier que la demande d'intégrer Darkino n'a pas déjà été faite 😉

**Comment by Tritri1029** [comment]
Alors oui effectivement je viens de voir que il y'avait déjà une demande je suis désolée.(vstream est vraiment trop bien bravo)

**Comment by sizanic** [comment]
Pas de souci.
Par contre, Darkino sera très dur à ajouter (a cause de la protection en place).

Tu trouves que vstream est vraiment trop bien, merci.
Mais en attendant l'ajout de darkino, savais-tu que vStream a mis en place une source de contenu EXCLUSIVE qui n'est pas hébergée sur un site. Tous les liens sont donc tous opérationnels, en plus d'être de grande qualité.

Cette source Premium est bien sûr proposée gratuitement par vStream, mais il faut toutefois posséder un compte vers un hébergeur ou un débrideur suivant : Uptobox, realDebrid ou alldebrid.

Il s'agit de la source Pastebin.

Par défaut cette source est vide, pour faire apparaitre le contenu, il faut saisir des codes.

L'équipe vStream ne propose pas de code.
Mais nous sommes partenaire avec la communauté "les alKODIques" qui propose tout un pack de codes.

Retrouves cette communauté sur : 
- facebook : https://www.facebook.com/groups/alkodiques
- Discord : https://discord.gg/3RTRvT2PSE
- le site : lesalkODIques.com

**Reply by Tritri1029** [reply]
Oui je viens de le faire justement et franchement j'ai cherché des trucs et j'ai tout trouvé c'est trop bien sourtout avec l'intégration des débrideurs.J'ai aussi une autre idée je sais pas si c'est déjà dispo mais ça serait bien de pouvoir avoir une liste de priorité pour les sources enfin par exemple si on utilise pastebin et wawaflix et faire en sorte que on veut que les résultats de pastebin apparaissent si disponible et si c'est indisponible sur pastebin utiliser wawaflix je sais pas si c'est très clair ?

**Reply by sizanic** [reply]
Mais a quoi ça servirait de chercher sur wawaflix ?
Car il me semble que cette source ne fonctionne pas sur vStream 🤔 

Par contre, il existe bien des moyens de lancer d'abord une recherche sur Pastebin, puis sur une autre source au besoin.

Le moyen simple : 
- désactiver toutes les sources sauf ces 2 là dans les paramètres de vStream/Sources/Activer les sources.

Le moyen moins simple : 
- utiliser l'extension tmdb et lui ajouter un "player" de recherche pour qu'elle recherche seulement dans Pastebin, et parametrer une source de recherche secondaire vers wawacity (ou autres sources laissées actives dans vStream).

Mais ceci est une autre histoire.
Les alkodiques en parle ici : https://lesalkodiques.com/docs/tmdb-helper/creer-son-propre-player-tmdb-helper-2/

**Reply by Tritri1029** [reply]
merci :)
